[IMP] inventory: CRL - replenishment & visibility days (#11444)
odoo/documentation: content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/lead_times.rst

> Lead times

Author: this is good feedback for improvement on the lead times doc, and it won't be covered in this PR (:

      :align: center
      :alt: Set security lead time for purchase from the Inventory > Configuration > Settings.

.. _inventory/warehouses_storage/days-to-purchase:

Days to purchase
----------------

> To set up the *days to purchase* lead time, go to :menuselection:`Inventory app --> Configuration

Author: ```suggestion
To set it up, go to :menuselection:`Inventory app --> Configuration
```
